app/src/androidTest/java/com/github/giommok/softwaredevproject/DatabaseTest.java

```diff
@@ -3,69 +3,61 @@
+    private static final String user1 = basicUsername + userOffset;
 
     /* Make sure that mock users are not on the database after a test */
     @After
     public void removeTestUsers() throws InterruptedException {
-        Database.refRoot.child("users").child("null").removeValue();
-        Thread.sleep(1500);
+        Database.refRoot.child(Database.CHILD_USERS).child(user1).removeValue();
+        Thread.sleep(sleepTime);
     }
 
     /* Test that isPresent method works */
     @Test
     public void isPresentTest() throws InterruptedException {
-        Database.setChild("users/null", Arrays.asList("username"), Arrays.asList("username@Test"));
-        Thread.sleep(1500);
-        Database.isPresent("users", "email", "dota2>lol", Assert::fail, () -> assertTrue("Correct behavior", true));
-        Database.isPresent("users", "username", "username@Test", () -> assertTrue("Correct behavior", true),  Assert::fail);
-        Thread.sleep(1500);
+        Database.setChild(Database.CHILD_USERS + user1, Collections.singletonList(Database.CHILD_USERNAME), Collections.singletonList(user1));
+        Thread.sleep(sleepTime);
+
+        Database.isPresent(Database.CHILD_USERS, Database.CHILD_USERNAME, "", Assert::fail, () -> assertTrue("Correct behavior", true));
+        Database.isPresent(Database.CHILD_USERS, Database.CHILD_USERNAME, user1, () -> assertTrue("Correct behavior", true),  Assert::fail);
+        Thread.sleep(sleepTime);
     }
 
     /* Test that setChild method works */
     @Test
     public void setChildTest() throws InterruptedException {
-        Database.isPresent("users", "username", "testing@Value", Assert::fail, () -> assertTrue("Correct behavior", true));
+        Database.isPresent(Database.CHILD_USERS, Database.CHILD_USERNAME, user1, Assert::fail, () -> assertTrue("Correct behavior", true));
         Thread.sleep(1500);
-        Database.setChild("users/null", Arrays.asList("username"), Arrays.asList("testing@Value"));
+        Database.setChild(Database.CHILD_USERS + user1, Collections.singletonList(Database.CHILD_USERNAME), Collections.singletonList(user1));
         Thread.sleep(1500);
-        Database.isPresent("users", "username", "testing@Value", () -> assertTrue("Correct behavior", true), Assert::fail);
+        Database.isPresent(Database.CHILD_USERS, Database.CHILD_USERNAME, user1, () -> assertTrue("Correct behavior", true), Assert::fail);
         Thread.sleep(1500);
     }
 
     /* Test that exception is thrown if different list sizes are provided */
-    @Test(expected = DatabaseException.class)
+    @Test(expected = RuntimeException.class)
     public void differentSizes_SetChildTest() throws InterruptedException {
-        Database.setChild("users/null", Arrays.asList("username", "error"), Arrays.asList("testingValue"));
+        Database.setChild(Database.CHILD_USERS + user1, Arrays.asList("string1", "string2"), Collections.singletonList(""));
         Thread.sleep(1000);

```

Comment:
Consider adding another constant (=1000)

Reply:
I removed this sleep as there was no need.